### ISSUE TITLE
runtime error

### ISSUE THREAD

**Issue by victor-moreno** [dpryan79/PileOMeth#23]
Hi,
I have compiled successfully in Centos 6.7, but get a runtime error when analyzing data.
During the first run, fasta index is built ok, then it fails. Next runs fails directly.

[error-message.txt](https://github.com/dpryan79/PileOMeth/files/252833/error-message.txt)

I have compiled it also on OSX, and the same problem. The error message is different, and mentions malloc problems:

PileOMeth(27055,0x7fff71bc7000) malloc: *** error for object 0x7ffe21d03188: incorrect checksum for freed object - object was probably modified after being freed.
*** set a breakpoint in malloc_error_break to debug
Abort trap: 6

**Comment by dpryan79** [comment]
That's a new one, can you past the command you're using?

**Comment by victor-moreno** [comment]
Hi,
I have aligned the data with last, marked duplicates with picard and converted to legacy-cigar with rtg sammerge. I used as reference human_g1k_v37_decoy.fasta

The call to PileOmeth is
ref=human_g1k_v37_decoy.fasta.gz
./PileOMeth/PileOMeth extract -r 21 -o pile/F1F $ref F1F.chr_21.bam

I have tested with a subset of just 50k reads and get the same error message. The difference is that I got some more data in bedGraph file. before was empty, now has:
